move-text.el

```diff
@@ -154,39 +158,38 @@ them when there's no region."
 ;;;###autoload
 (defun move-text-region-up (start end n)
   "Move the current region (START END) up by N lines."
-  (interactive (move-text-get-region-and-prefix))
-  (move-text-region start end (if (null n) -1 (- n))))
+  ;; (interactive (move-text-get-region-and-prefix))
+  (move-text-region start end -n))
 
 ;;;###autoload
 (defun move-text-region-down (start end n)
   "Move the current region (START END) down by N lines."
-  (interactive (move-text-get-region-and-prefix))
-  (move-text-region start end (if (null n) 1 n)))
+  ;; (interactive (move-text-get-region-and-prefix))
+  (move-text-region start end n))
 
 ;;;###autoload
-(defun move-text-up (&optional start end n)
+(defun move-text-up (start end n)
   "Move the line or region (START END) up by N lines."
   (interactive (move-text-get-region-and-prefix))
   (if (not (move-text--at-first-line-p))
     (if (region-active-p)
         (move-text-region-up start end n)
-      (if n (cl-loop repeat n do (move-text-line-up))
-        (move-text-line-up)))))
+      (cl-loop repeat n do (move-text-line-up)))))
 
 ;;;###autoload
-(defun move-text-down (&optional start end n)
+(defun move-text-down (start end n)
   "Move the line or region (START END) down by N lines."
   (interactive (move-text-get-region-and-prefix))
   (if (region-active-p)
       (move-text-region-down start end n)
-    (if n (cl-loop repeat n do (move-text-line-down))
-      (move-text-line-down))))
+    (cl-loop repeat n do (move-text-line-down))))
 
 ;;;###autoload
 (defun move-text-default-bindings ()
-  "Use default bindings for move-text-up and move-text-down (M-up / M-down)."
+  "Use default bindings for move-text-up and move-text-down (M-up / M-down).
+
+Bind `move-text-up' and `move-text-down' to M-up & M-down."
   (interactive)
```

Comment:
Seems like this is repetition, pick one and ... ok the original text already says this.  

`Tick Quoting' the fn names is a good call though.

Reply:
```
"Use default bindings for `move-text-up' and `move-text-down' (M-up / M-down)."
```